docs/src/pages/reference/QueryClient.md

```diff
@@ -88,7 +88,7 @@ try {
 
 **Options**
 
-The options for `fetchQuery` are exactly the same as those of [`useQuery`](#usequery).
+The options for `fetchQuery` are exactly the same as those of [`useQuery`](#usequery), except the following: `enabled, refetchInterval, refetchIntervalInBackground, refetchOnWindowFocus, refetchOnReconnect, notifyOnChangeProps, notifyOnChangePropsExclusions, onSuccess, onError, onSettled, useErrorBoundary, select, suspense, keepPreviousData, placeholderData`; which are stictly for useQuery and useInfiniteQuery. You can check the [source code](https://github.com/tannerlinsley/react-query/blob/361935a12cec6f36d0bd6ba12e84136c405047c5/src/core/types.ts#L83) for more clarity.
```

Comment:
- the link to `useQuery` is not working for me - can you maybe fix that, too?

- if we are linking to the source, should we link to the `master` branch? I usually like permalinks, but it will be outdated if we add more things

Reply:
I thought of linking to master but what if the content gets outdated as you say? That's why i linked it to a recent commit.

Reply:
I made the change and it looks good https://react-query-ifwq37wlh.vercel.app/reference/QueryClient#queryclientfetchquery